main.tf

```diff
@@ -425,3 +425,23 @@ resource "aws_default_vpc" "this" {
 
   tags = "${merge(map("Name", format("%s", var.default_vpc_name)), var.default_vpc_tags, var.tags)}"
 }
+
+data "aws_route_table" "default" {
```

Comment:
Why use a data source ?
Route table ID is available by `concat(aws_vpc.this.*.default_route_table_id, [""])[0]` (see outputs)